cob_cartesian_controller/action/CartesianController.action

```diff
@@ -11,3 +10,5 @@ bool success
 string message
 ---
 # Define a feedback message
+bool success
```

Comment:
The Server currently doesn't send any feedback (and also the client does not subscribe to it...) but you are right: 
```
uint8 state
string message
``` 
would probably be better